Sources/ShopifyCheckoutSheetKit/CheckoutBridge.swift

```diff
@@ -101,8 +102,13 @@ extension CheckoutBridge {
 			switch name {
 			case "completed":
 				let checkoutCompletedEventDecoder = CheckoutCompletedEventDecoder()
-				let checkoutCompletedEvent = try checkoutCompletedEventDecoder.decode(from: container, using: decoder)
-				self = .checkoutComplete(event: checkoutCompletedEvent)
+				do {
+					let checkoutCompletedEvent = try checkoutCompletedEventDecoder.decode(from: container, using: decoder)
+					self = .checkoutComplete(event: checkoutCompletedEvent)
+				} catch {
+					os_log(.error, "[ShopifyCheckoutSheetKit] Error decoding CheckoutCompletedEvent: %{public}@", error as CVarArg)
```

Comment:
@kiftio when this happens, we know we have a problem. Do you think it's worth adding a message here to say "Please open an issue in https://github.com/Shopify/checkout-sheet-kit-swift"?

Reply:
Yeah, I think so. Should we also add a test for this to check the message that's logged?